src/App.js

```diff
@@ -1,5 +1,5 @@
 import React, { Component } from 'react';
-import { BrowserRouter as Router, Switch, Route, Redirect } from 'react-router-dom';
+import { BrowserRouter as Router, Redirect, Route, Switch } from 'react-router-dom';
```

Comment:
Alphabetical? lol that is PICKY! :-P

Reply:
somehow `prettier` works slightly differently when used from IntelliJ :man_shrugging: